lib/collection/response.js

```diff
@@ -11,6 +11,7 @@ var util = require('../util'),
     Header = require('./header').Header,
     CookieList = require('./cookie-list').CookieList,
     HeaderList = require('./header-list').HeaderList,
+    contentInfo = require('../content-info/index'),
```

Comment:
`contentInfo = require('../content-info/index').contentInfo`

Reply:
`/index` can be omitted